internal/dot/README.md

```diff
@@ -0,0 +1,64 @@
+# Dot
+
+The dot module generates a DOT file representation of a dependency graph.
+
+## Interpreting the graph
+
+The graph should be read from left to right. The leftmost node in the graph (the root node) depends
+on its dependency tree to the right. An arrow from node_a to node_b in the graph means that node_b
+is consumed by node_a and that node_b is a parameter of node_a.
+
+**Nodes:**
```

Comment:
I actually like my current description because understanding the direction and flow of the graph was one of the more difficult tasks for me over the course of this project.

Reply:
Fair enough. I think I was unsure about this. (Nit: No need for ":" in headers.)